vispy/visuals/markers.py

```diff
@@ -651,7 +557,7 @@ def __init__(self, symbol='o', scaling=False, alpha=1, antialias=1, spherical=Fa
 
     def set_data(self, pos=None, size=10., edge_width=1., edge_width_rel=None,
                  edge_color='black', face_color='white',
-                 symbol=None, scaling=None):
```

Comment:
@brisvag Was removing `scaling` kwarg here intentional?
It breaks the example here for the macOS OpenGL issue.
https://github.com/vispy/vispy/issues/2078#issuecomment-863586236

Reply:
Yes; the idea was to unify the behaviour such as:
- uniforms are just properties with setters (no data needs to be sent to the GPU in buffers)
- `set_data` only takes stuff that needs to be sent as uniforms

Maybe I should have done it in a separate PR and more explicitly, at the time it just seemed like a small refactor that made things cleaner ^^'